bounties/npm/@hapi/1/vulnerability.json

```diff
@@ -0,0 +1,52 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-05",
+    "AffectedVersionRange": "19.1.1",
```

Comment:
Can you please confirm the affected range. Generally, we only accept disclosures for vulnerabilities that affect all (`*`) versions and or the latest version.

Reply:
Yes. this is the latest version - 19.1.1